controllers/storagecluster/cephcluster.go

```diff
@@ -70,6 +70,19 @@ func arbiterEnabled(sc *ocsv1.StorageCluster) bool {
 	return sc.Spec.Arbiter.Enable
 }
 
+func verifyMonitoringEndpointsOfCephCluster(cephCluster *rookCephv1.CephCluster, log logr.Logger) error {
+	if cephCluster == nil {
+		return fmt.Errorf("empty ceph cluster passed as argument")
+	}
+	monIPs := make([]string, len(cephCluster.Spec.Monitoring.ExternalMgrEndpoints))
+	for idx, endpointAddress := range cephCluster.Spec.Monitoring.ExternalMgrEndpoints {
+		monIPs[idx] = endpointAddress.IP
+	}
+	monIPStr := strings.Join(monIPs, ",")
+	monPort := fmt.Sprint(cephCluster.Spec.Monitoring.ExternalMgrPrometheusPort)
```

Comment:
Is there really a need for an additional util function that is called only once from a single call site?
Can't we just embed the code in the call site or alternatively add this code to `verifyMonitringEndpoints` (which I believe is also called only once from a single call site)

Reply:
I really agree. I was trying to add a very minimal change (and mostly trying not to touch the existing code).
We can definitely add this new logic into the existing `verifyMonitringEndpoints()` function.

Reply:
Refactor is a refactor, we do not need to be afraid of changes, we have a review process in place and strong QE. 
I will argue that avoiding touching existing code will always create much more convoluted code over a long period of time which will make every future change even harder to reason about

Reply:
Ack and I agree.
There were times when comments were like this =)
```We can do all the refactoring later. Add just the minimal code to solve the issue or else it will be very hard to say whether we have introduced a new bug (or not)```